target/snitch_cluster/Makefile

```diff
@@ -218,12 +232,30 @@ test/bootrom.elf test/bootrom.dump test/bootrom.bin: test/bootrom.S test/bootrom
 	riscv64-unknown-elf-objcopy -j .text -O binary test/bootrom.elf test/bootrom.bin
 
 
-$(SNAX_GEN):
-	${WRAPPER_GEN} --cfg_path="$(ROOT)/target/snitch_cluster/$(CFG_OVERRIDE)" \
+SNAX_TEMPLATE_SOURCES = $(wildcard $(SNAX_TPL_PATH)*.tpl)
```

Comment:
Is this actually doing something? I mean it just looks like it listed the files but we're not using it in our wrapper gen. Because our wrapper gen loads this inside already.

Reply:
Or is it like just ensuring that the files are there?

Reply:
The files generated by the wrappergen are dependent on these files, so they need to be put there such that make can correctly regenerate them if they are altered, or it can throw an error if they are not there.
I would advise to always put dependencies explicitely, since the reason this PR exists in the first place is because file dependencies were not properly listed before. It also makes it easier to figure out what is going on if you are reading the makefile.